dvc/fs/dvc.py

```diff
@@ -238,7 +272,8 @@ def _open(
         dvc_path = _get_dvc_path(dvc_fs, subkey)
         return dvc_fs.open(dvc_path, mode=mode)
 
-    def isdvc(self, path, **kwargs):
+    def isdvc(self, path, **kwargs) -> bool:
+        """Is this entry dvc-tracked?"""
```

Comment:
fsspec is styled in the same way, see [`fs.isfile`](https://github.com/fsspec/filesystem_spec/blob/742a103849adf2d0c9f3ad3fe08010d8de688268/fsspec/spec.py#L646) and [`fs.isdir`](https://github.com/fsspec/filesystem_spec/blob/742a103849adf2d0c9f3ad3fe08010d8de688268/fsspec/spec.py#L646).